src/libsync/CMakeLists.txt

```diff
@@ -1,4 +1,14 @@
-find_package(LibreGraphAPI REQUIRED)
+set(LibreGraphAPIVersion v0.11.0)
```

Comment:
In the future we should also pass that version to find_package, however, at the moment the package does not provide its version.